mne/channels/channels.py

```diff
@@ -127,10 +125,10 @@ def equalize_channels(candidates, verbose=None):
     from ..evoked import Evoked
     from ..time_frequency import AverageTFR
 
-    if not all(isinstance(c, (BaseRaw, BaseEpochs, Evoked, AverageTFR))
-               for c in candidates):
-        raise ValueError('candidates must be Raw, Epochs, Evoked, or '
-                         'AverageTFR')
+    for candidate in candidates:
+        _validate_type("Instances to be merked", candidate,
```

Comment:
marked

Reply:
what the hell is "merked"

Reply:
apparently, "merk" is an obsolete spelling of "mark", so you are just out of date :)

Reply:
I think I wanted to write "merge".

Reply:
Ahh yes I thought for some reason this was with ICA changes, that makes more sense

Reply:
I don't think "merge" makes that much sense actually :)